extmod/network_wiznet5k.c

```diff
@@ -896,11 +904,11 @@ STATIC mp_obj_t wiznet5k_ifconfig(size_t n_args, const mp_obj_t *args) {
         self->netinfo.dhcp = NETINFO_STATIC;
         mp_obj_t *items;
         mp_obj_get_array_fixed_n(args[1], 4, &items);
-        netutils_parse_ipv4_addr(items[0], netinfo.ip, NETUTILS_BIG);
-        netutils_parse_ipv4_addr(items[1], netinfo.sn, NETUTILS_BIG);
-        netutils_parse_ipv4_addr(items[2], netinfo.gw, NETUTILS_BIG);
-        netutils_parse_ipv4_addr(items[3], netinfo.dns, NETUTILS_BIG);
-        ctlnetwork(CN_SET_NETINFO, &netinfo);
+        netutils_parse_ipv4_addr(items[0], self->netinfo.ip, NETUTILS_BIG);
```

Comment:
Can you explain this change. I think it makes sense, but it would be good to clarify -- is self->netinfo only for DHCP or is it correct to also put the static address into it? What goes wrong if you don't make this change?